src/Templates/src/templates/maui-contentpage-csharp/.template.config/ide.host.json

```
@@ -0,0 +1,7 @@
+{
+    "$schema": "http://json.schemastore.org/vs-2017.3.host",
+    "appliesTo": "UseMaui",
+    "defaultItemExtension": "cs",
+    "itemHierarchyPaths": [ ".NET MAUI" ],
+    "icon": "ide/MauiPageCSFile.ico"
+}    
```

```suggestion
}
```

Removing the niggling whitespace at the end 😁